api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/TestAppender.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestAppender extends AppenderBase<ILoggingEvent> {
```

Comment:
Might it be worthwhile putting this into htbhf-common-test now that we're reusing it?

Reply:
Done.